sdk/eventhub/azure-eventhub-checkpointstoreblob-aio/README.md

```diff
@@ -58,15 +58,15 @@ The easiest way to create a `ContainerClient` is to use a connection string.
 from azure.storage.blob.aio import ContainerClient
 container_client = ContainerClient.from_connection_string("my_storageacount_connection_string", "mycontainer")
 ```
-For other ways of creating a `ContainerClient`, go to [Blob Storage library](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/storage/azure-storage-blob) for more details.
+For other ways of creating a `ContainerClient`, go to [Blob Storage library](../../storage/azure-storage-blob) for more details.
 
 ### Create an `EventHubConsumerClient`
 The easiest way to create a `EventHubConsumerClient` is to use a connection string.
 ```python
 from azure.eventhub.aio import EventHubConsumerClient
 eventhub_client = EventHubConsumerClient.from_connection_string("my_eventhub_namespace_connection_string", event_hub_path="myeventhub")
 ```
-For other ways of creating a `EventHubConsumerClient`, refer to [EventHubs library](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/eventhub/azure-eventhubs) for more details.
+For other ways of creating a `EventHubConsumerClient`, refer to [EventHubs library](../azure-eventhub) for more details.
```

Comment:
PyPI link as well

Reply:
Are you suggesting that we should change this from a source link to a link the the package on pypi?

Reply:
I think he's suggesting that by using a relative URL here it wont work once this readme is published to PyPI. So we should use complete URLs.

Reply:
Actually - I stand corrected - links to other packages should probably be to their PyPI reference rather than their source.
However my comment still stands - we can't use relative URLs in the readme as they wont work once this package is published.

Reply:
@annatisch @lmazuel Please see the comments at https://github.com/Azure/azure-sdk-for-python/pull/8813#discussion_r349801308, the relative links will get correctly translated to a proper absolute link before we publish so they are the only way to get stable links to the source tree. 

As for the pypi links vs source links that is a different question and if we feel we want to switch them I think the eventhubs owner should do that. My main clean-up here is to correct the incorrect links that are currently pointing to master.

Reply:
fair enough :)